packages/amplify-util-uibuilder/src/__tests__/generateComponents.test.ts

```diff
@@ -37,15 +40,23 @@ describe('can generate components', () => {
           name: 'testSchema',
           schemaVersion: '1.0',
         },
+        {
+          resultType: 'FAILURE',
+          schemaName: 'testSchema',
+          name: 'testSchema',
+          schemaVersion: '1.0',
+          schema: { id: 'f-123456' },
+        },
       ],
     };
     mockedExport = jest.fn().mockReturnValue({
       entities: schemas.entities,
     });
-    awsMock.AmplifyUIBuilder = jest.fn().mockReturnValue({
-      exportComponents: jest.fn().mockReturnValue({
+    mockedDeleteForm = jest.fn(() => true);
+    awsMock.AmplifyUIBuilder = jest.fn(() => ({
+      exportComponents: jest.fn(() => ({
```

Comment:
This is fine as is, but as an FYI `jest.fn().mockResolvedValue` is helpful syntactic sugar: https://jestjs.io/docs/mock-function-api#mockfnmockresolvedvaluevalue